src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/ZLibException.cs

```diff
@@ -25,7 +25,7 @@ public class ZLibException : IOException, ISerializable
         /// <param name="zlibErrorContext">A description of the context within zlib where the error occurred (e.g. the function name).</param>
         /// <param name="zlibErrorCode">The error code returned by a ZLib function that caused this exception.</param>
         /// <param name="zlibErrorMessage">The string provided by ZLib as error information (unlocalised).</param>
-        public ZLibException(string message, string zlibErrorContext, int zlibErrorCode, string zlibErrorMessage) : base(message)
+        public ZLibException(string message, string zlibErrorContext, int zlibErrorCode, string? zlibErrorMessage) : base(message)
```

Comment:
Both message and zlibErrorContext should be nullable, no?

Reply:
yes